packs/bit-bucket/actions/create_ssh_key.yaml

```diff
@@ -0,0 +1,18 @@
+name: create_ssh_key
+runner_type: run-python
+description: Create SSH key
+enabled: true
+entry_point: create_ssh_key.py
+parameters:
+  repo:
+    type: string
+    description: Requires repo name
+    required: true
+  ssh_key:
+    type: string
+    description: Generated ssh-key
```

Comment:
Would be better to say something like "Public SSH key to associate with your account".